Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:2.7-alpine
+RUN apk add --update git openssl
+WORKDIR /src
+COPY requirements.txt /src
+RUN pip install -r requirements.txt
+COPY . /src
+CMD pytest --driver=SauceLabs \
+    --junit-xml=results/py27.xml \
+    --html=results/py27.html --self-contained-html \
+    --log-raw=results/py27_raw.txt \
+    --log-tbpl=results/py27_tbpl.txt \
```

Comment:
The docker command doesn't need the various results files. Also, I'd like it if by default the tests ran within the docker container using Firefox so that we're not limited to using Sauce Labs. The open pull request for FxAPOM includes GeckoDriver and Firefox, but I've not yet tried these with alpine linux.

Reply:
Ooops; results-files spew fixed.  I'll leave the test-running issue to a future iteration (will likely need to work with you on it).